mantis-control-plane/mantis-control-plane-client/src/main/java/io/mantisrx/server/master/client/config/WorkerConfigurationUtils.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2022 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.mantisrx.server.master.client.config;
+
+import io.mantisrx.server.core.MetricsCoercer;
+import java.util.Properties;
+import org.skife.config.ConfigurationObjectFactory;
+
+public class WorkerConfigurationUtils {
```

Comment:
I'm guessing this library needs to live in TaskExecutor module.

Reply:
It's also needed on nfmantis worker titus. Let me see if I can move this one to runtime-loader instead.

Reply:
any updates on this?